### ISSUE TITLE
Request: allow / instead of /resources

### ISSUE THREAD

**Issue by JulianLeviston** [bhauman/lein-figwheel#111]
It'd be very handy if I could set figwheel's compilation path to /app/assets/javascripts/ rather than it requiring /resources as a prefix.

I'm assuming there's a really good reason why you put that constraint in. Is it very difficult to relax that constaint but leave the /resources folder as an over-ridable default?

I'm using figwheel to (re)build a Reagent app from an Om app that was an Ember app which is all sitting on a Rails backend. (groan) yes, Rails. :(

So, in order to have my css and javascript files, and the CSRF protection stuff working, I need to serve my files out of the /app/assets/ folder. In the meantime, I'm going to rename my assets folder and point rails at that instead of its default location, but I'd prefer not to.

**Comment by bhauman** [comment]
If you are using Rails to serve your application you do not need to have
your assets in resources. You can have them wherever you want to put them.
There is only a requirement if you are serving your app with the figwheel
server.

Sorry that this currently isn't clear in the documentation.

**Comment by JulianLeviston** [comment]
Turns out I was able to just add the resources folder to the rails app's asset pipleline and modify my javascript tags to asset tags.

Odd... figwheel complained if I didn't put the files in resources, and it kept reloading half-compiled files... I'm not entirely sure why. It still does it every now and then... perhaps it's the asset pipeline lagging because of compilation and figwheel thinking the files are there but they're not actually. Guh Rails. I'm going to try and get off it as soon as I possibly can.
